services/web/server/src/simcore_service_webserver/login/handlers_registration.py

```diff
@@ -259,6 +270,12 @@ async def register(request: web.Request):
 
     # NOTE: Here confirmation is disabled
     assert settings.LOGIN_REGISTRATION_CONFIRMATION_REQUIRED is False  # nosec
+    assert (  # nosec
```

Comment:
Why is it being checked again here?

Reply:
these are all for "documentation". 
This part of the code will be refactored soon and want to make sure that all the components are "well documented" with asserts.